tests/datasource/data_connector/test_data_connector.py

```diff
@@ -53,77 +52,6 @@ def basic_data_connector(tmp_path_factory):
     return basic_data_connector
 
 
-# TODO: <Alex>This test should be moved to "tests/datasource/data_connector/test_configured_asset_filesystem_data_connector.py".</Alex>
```

Comment:
Why remove these tests?

Reply:
These were actually moved to `test_configured_asset_filesystem_data_connector`